src/App.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Title from './TitleComponent/TitleComponent'
+import './App.css'
+class App extends React.Component {
+
+    constructor(props){
+        super(props);
+        this.state = {
+            content : '',
+            name: '',
+            number: '',
+        }
+    }
+
+    inputHandler(event) {
```

Comment:
According to naming convention of handlers function you should rename this function to handleChange (as we pass it to onChange)